sql-cli/sql_cli/connections.py

```diff
@@ -31,7 +19,8 @@ def validate_connections(connections: list[Connection], connection_id: str | Non
     for connection in connections:
         if connection.id == connection_id:
             config_file_contains_connection = True
-        _create_or_replace_connection(connection)
+        # Create connection using Environment Variable
+        os.environ[f"AIRFLOW_CONN_{connection.conn_id.upper()}"] = connection.get_uri()
```

Comment:
@kaxil maybe we can have a way to convert the yaml in the test-connections.yaml into these URIs? the test-connections.yaml is very useful it would be a bummer to lose it.

Reply:
hmm, how does this impact test-connections.yaml?

Reply:
I understand that we're only using env vars internally - the file continues to be used by the end users.